README.md

```diff
@@ -1,2 +1,2 @@
 # Demo Project
```

Comment:
This is an in line comment (at `Line 1`) with some **fancy** `markdown` and embedded xml (code highlight)
```xml
  <some>
    <embedded x="ml"/>
    Crazy stuff
  </some>
```